.rubocop.yml

```diff
@@ -8,7 +8,6 @@ AllCops:
   Exclude:
     - 'vendor/**/*'
     - 'spec/internal/bin/*'
```

Comment:
I'll bet that `spec/internal` is entirely gone since we've upgraded rubocop.  Want to remove that line too?